package.json

```diff
@@ -2,9 +2,9 @@
   "name": "prettier-setup",
   "license": "MIT",
   "devDependencies": {
+    "eslint": "4.15.0",
```

Comment:
Didn't mean to change this myself, looks like Prettier decided to rearrange things…